### ISSUE TITLE
On switched language, Static Page Repeater field data doesn't removed after the field is deleted.

### ISSUE THREAD

**Issue by haralake** [rainlab/translate-plugin#492]
OctoberCMS Build: ### 455
PHP Version: 7.2
Database Engine: MySQL
Plugins Installed: Rainlab.StaticPages, Rainlab.Translate

Description:
I'm using a simple repeater variable on static page:
```
{repeater name="content_sections" prompt="Add another content section"}
<h3>
{text name="content_header" label="Content section" placeholder="Type in a heading and enter some content for it below"}{/text}
</h3>
<div>
{richeditor name="content_body" size="large"}{/richeditor}
</div>
{/repeater}
```   

On static pages backend i create one static page, then i switch language and using repeater field i create 1 field and i save page. If i delete this field and then save again, data doesn't removed as supposed . You can see this clearly on sample video. I test it on a clean install with latest build.

![Screenshot](http://iasmoscosmos.gr/images/octobercms/repeater-language-issue.gif)

**Steps To Reproduce:**
1. Create one repeater variable on static page layout
2. Create one static page
3. On static page switch language with language selector
4. Create one field using repeater field and save page
5. Delete this field you have created and save page
6. On static page content that keeps fields data you will see that data are still there, has not been deleted.

**Comment by bennothommo** [comment]
This will likely be resolved once a fix is deployed for https://github.com/octobercms/october/issues/4402.

**Comment by haralake** [comment]
@bennothommo i think this is a different issue that has to do with language plugin. This is the reason i created a new issue on Translate plugin. The specific issue only happens when you switch language. Of course you know better and maybe these problems have the same cause.

**Comment by ketihar** [comment]
I confirm this specific bug. On static page when i switch language i can create data using a repeater field, but i can't delete them. On backend i can see that have successfully deleted but when i close and reopen static page data are still there.

**Comment by mjauvin** [comment]
l{Looks like this is only happening if there is no repeater field created for the default language

**Comment by haralake** [comment]
I also test it with and without repeater field on default language, i get the same issue. This behavior is not happened on default language.

**Comment by bennothommo** [comment]
@haralake Thanks for the further info. I will investigate more and will let you know if I find anything.

**Comment by daftspunk** [comment]
This has been fixed in October CMS v3.1 by introducing a global language picker and simplifying the code used by this plugin.

- Link to the [upgrade guide](https://github.com/rainlab/translate-plugin/blob/master/UPGRADE.md)
- Link to [multisite documentation](https://docs.octobercms.com/3.x/cms/resources/multisite.html)

If you still need help, feel free to get in touch with the support desk for assistance: https://octobercms.com/contact

Thank you
